deployment/kubernetes/charts/origin/templates/discovery.ingress.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
     certmanager.k8s.io/cluster-issuer: {{ .Values.clusterIssuer }}
     nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
     nginx.ingress.kubernetes.io/rewrite-target: /graphql
+    nginx.ingress.kubernetes.io/limit-rps: "5"
```

Comment:
That's low. On the browse/search page the DApp will issue 1 request per listing card and we have 12 cards. Based on that, I'd set this to something in the order of 24.

Reply:
Thanks, I didn't consider the retrieving listings bit.